packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -213,9 +213,6 @@ class BondWithdrawalWatcher extends BaseWatcher {
             this.bridge.formatUnits(bondedAmount)
           )
         })
-        .catch(async (err: Error) => {
```

Comment:
Out of curiosity, why was this removed -- is it redundant? 

My understanding is that our new change will not throw anymore, but what do you expect to happen if another part of the code throws and this does not exist?

Reply:
yes it's redundant. Removing catch that throws same error produces same result

```
async function p() {
  throw new Error('not ok error')
}

async function main() {
  try {
    await p().then(() => console.log('ok'))
    // removing this has same effect
    .catch(err => {
      throw err
    })
  } catch (err) {
    console.log('got error:', err)
  }
}

main()
```